pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -351,15 +358,12 @@ func (o *Options) Run() error {
 		return nil
 
 	default:
-		if c := findClusterOperatorStatusCondition(cv.Status.Conditions, configv1.OperatorDegraded); c != nil && c.Status == configv1.ConditionTrue {
-			prefix := "No upgrade is possible due to an error"
-			if c := findClusterOperatorStatusCondition(cv.Status.Conditions, configv1.OperatorProgressing); c != nil && c.Status == configv1.ConditionTrue && len(c.Message) > 0 {
-				prefix = c.Message
+		if c := findClusterOperatorStatusCondition(cv.Status.Conditions, clusterStatusFailing); c != nil && c.Status == configv1.ConditionTrue {
+			if c.Status != configv1.ConditionFalse {
```

Comment:
The condition check seems not to be duplicate with the condition in line#361 `c != nil && c.Status == configv1.ConditionTrue`.

Reply:
Thanks :).  Rerolled in f95fea08d -> 33c93e15c to drop the `c.Status == configv1.ConditionTrue` guard.